package.json

```diff
@@ -16,10 +16,12 @@
     }
   },
   "engines": {
-    "node": "8.11.4",
-    "yarn": "1.9.4"
+    "node": "^8.11.1",
```

Comment:
please leave the engine specification as-is, you can change it in your fork :) But we need it for the installs on heroku.

Reply:
OK